meta-openpli/recipes-multimedia/tuxtxt/tuxtxt-enigma2.bb

```diff
@@ -8,6 +8,7 @@ inherit gitpkgv
 
 GITHUB_URI ?= "git://github.com"
 SRC_URI = "${GITHUB_URI}/OpenPLi/tuxtxt.git"
+SRC_URI_append_libc-musl = " file://0001-tuxtxt-enigma2-fix-build-with-musl.patch"
```

Comment:
@andrea-adami Would you please send a PR for https://github.com/OpenPLi/tuxtxt or @littlesat  could apply this patch.